index.html

```diff
@@ -249,23 +249,26 @@ <h3>Partnerships</h3>
                             <div class="floe-front-news">
                                 <h3>News</h3>
 
+                                <h4>Introducing UI Options Plus (UIO+)</h4>
+                                <time class="floe-date" datetime="2018-01-31">January 31, 2018</time>
+                                <p>
+                                    We have recently published a beta of <a href="https://chrome.google.com/webstore/detail/ui-options-plus-uio+/okenndailhmikjjfcnmolpaefecbpaek">User Interface Options Plus (UIO+)</a> to the Chrome Web Store.
+                                    UIO+ is a new tool to enable the personalization experience of UI Options as a browser extension.
+                                </p>
+                                <p><a href="./news/2017-07-19-inclusive-science-lab.html">Read more: Introducing UI Options Plus (UIO+)</a></p>
```

Comment:
Is this link correct? It seems to be pointing to a file called <code>2017-07-19-inclusive-science-lab.html</code>, but shouldn't it link to <code>2018-01-31-uioPlus.html</code>?